frontend/invest/src/styles/tokens.css

```
@@ -0,0 +1,222 @@
+   Auto Trader Invest — Foundations
+   Toss-style Korean fintech: light surfaces, calm grayscale,
+   confident blue accent, Korean-fintech P/L color rule
+   (▲ red = gain, ▼ blue = loss).
+   ========================================================= */
+
+/* ---------- Webfont: Pretendard (Korean + Latin) ----------
+   Pretendard is the de-facto Korean fintech body face used by
+   Toss, Naver, Kakao products. Loaded from cdn.jsdelivr.net.
+   Numbers: tabular-nums + ss06 for clean fintech digits.
+   ---------------------------------------------------------- */
+@import url("https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.css");
```

Stylelint (17.10.0)

[error] Expected "url("https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.css")" to be ""https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.css"" (import-notation)

(import-notation)